docs/source/advanced.rst

```diff
@@ -1,6 +1,6 @@
-===================================
-Advanced Usage of SciKeras Wrappers
-===================================
+==============
+Advanced Usage
```

Comment:
Why is the section on loss selection in "advanced usage"? I suspect the average Keras user knows about loss/optimization.

Reply:
That is what it was historically named... I only shortened the name here in an attempt to get links working (and because the `of SciKeras Wrappers` part is arguably redundant). I'm happy to change it, in another PR.